src/lib/ListMetrics.js

```diff
@@ -200,7 +199,7 @@ export default class ListMetrics {
                         cmd: key,
                     });
                 } else {
-                    const m = getMetricFromKey(key, resource, this.metric);
```

Comment:
This method has been updated in a prior PR, hence the removal of these parameters.